sim/main.cpp

```diff
@@ -90,7 +93,7 @@ int main(int argc, char **argv)
 
         // Create the simulator
         sparta::Scheduler scheduler;
-        OlympiaSim sim("core_topology_1",
+        OlympiaSim sim("core_topology_simple",
```

Comment:
I'm lazy -- I like fewer words ...
```suggestion
        OlympiaSim sim("simple",
```